script/functions/apiUtils.js

```diff
@@ -0,0 +1,60 @@
+import { goToFolderhtml } from '../temporary.js';
+import { getTokenInLocalStorage, saveTokenInLocalStorage } from './token.js';
+
+const apiUrl = 'https://bootcamp-api.codeit.kr/docs';

```

Comment:
요게 바로 본문에 적어주신 cors 에러가 나는 이유네요 ㅎㅎ baseUrl이 잘못 설정되어있어요.
```suggestion
const apiUrl = 'https://bootcamp-api.codeit.kr/api';

```
요렇게 변경해줘야할 것 같아요.

Reply:
Commit ID : 7ee23f2d1d1ebe463b8e28a40aba171413c084d8
고쳤습니다.. 제대로 체크를 못했네요.. :(